src/state/reducer.ts

```diff
@@ -13,6 +13,13 @@ type Writable<T> = {
   [P in keyof T]: NonNullable<T[P]>;
 }
 
+function resourceCustomizer(a: any, b: any): any {
+  if (a instanceof Resource && b instanceof Resource) {
+    const Static = b.constructor as typeof Resource;
+    return Static.merge(a, b);
+  }
+}
```

Comment:
can you just return the second one as fallback?

Reply:
also this needs a test case

Reply:
https://lodash.com/docs/#mergeWith

By returning undefined, `lodash.merge()` will be used. This is necessary to make sure that the `entities` object itself is merged properly.

But I can make it explicitly return undefined, and add a comment to explain it.

Reply:
Ah didn't realize that. I think a comment is sufficient no need for the explicit return.